app/helpers/user_to_project_notification.py

```diff
@@ -1,10 +1,16 @@
 from flask import render_template
 from .email import send_email
+import os
+from flask_restful import Resource, request
```

Comment:
Maintain basic flask to avoid mixing with flask restful. Same implementation can be refactored using basic flask for the URL being constructed

Reply:
remove unused imports